src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
+  background-color: #ddd;
+  color: black;
   margin: 0;
+  min-width: 1024px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
```

Comment:
it's better to set it for elements you really need instead of setting it for entire page

Reply:
You’re right. Done!